README.md

```diff
@@ -101,6 +101,20 @@ IP/port of the Istio Ingress gateway.
     curl --user bob:password -i http://$GATEWAY_URL/api/v1/products
     ```
 
+## How to use the OPA-Istio plugin
+
+### Option 1: Using `vendored` OPA dependency
```

Comment:
Do we have a scenario where this option makes sense? I'm wondering if we should only recommend (1) running the opa_istio binary/container produced by the makefile or (2) building the .so file with the GOPATH option and then running with either (i) upstream OPA (of the correct version) or (ii) a custom OPA executable.